app/src/main/java/com/example/mona/facebookoffline/EditActivity.java

```diff
@@ -0,0 +1,23 @@
+package com.example.mona.facebookoffline;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+public class EditActivity extends Activity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_edit);
+
+        Button save_button = (Button) findViewById(R.id.save_button);
```

Comment:
mega-nit: Java variable names should be camelCase...even though xml names tend to use underscores. It's weird <_<;